schema.sql

```diff
@@ -1,5 +1,9 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id int,
```

Comment:
- [ ] [OPTIONAL] Great implementation 👍🏽 However, we can also make the `id` column not null and autoincremented
`id int NOT NULL IDENTITY(1,1)`

Reply:
thanks a lot for this.